src/VisualStudio/Core/Impl/Options/GridOptionPreviewControl.xaml.cs

```diff
@@ -112,6 +113,8 @@ internal override void Close()
 
         internal void Generate_Save_EditorConfig(object sender, System.Windows.RoutedEventArgs e)
         {
+            Logger.Log(FunctionId.ToolsOptions_GenerateEditorconfig);
+
```

Comment:
Do we also want to log an additional event just after user saves the file? I'd let @mikadumont decide if we need 2 telemetry events (click + save) versus 1 (click).

Reply:
Do we currently track when a user adds an editorconfig from the add new item template as well as the add new file template? If so having telem on save can be useful to get an overall number on how many editorconfig files are added to a given project/sln.

Reply:
Hmm, I'm not sure if we have telemetry when a user adds a new editorconfig from the add new item template, although I'm leaning towards no since I can't find any applicable FunctionIds. (@mavasani please correct me if I'm wrong.)

Also, if we wanted to get an overall # of how many editorconfig files have been added to a given project/solution, would we also need to add telemetry to the Configure severity code action since an .editorconfig could be generated that way as well? @mikadumont 

I asked Mika last week regarding the 2 telemetry events (click + save) versus 1 (click), and I believe she indicated that the initial idea was telemetry just for when the user clicks the button.